modules/add.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+/* eslint-disable */
```

Comment:
- [ ] Based on the Microverse project best practices, you are not allowed to `disable` linters no matter what, we encourage you to fine solutions to whatever errors that may be raised. This is usually the case in a real work settings, we are being trained for the workplace. Linters are put in place to help avoid code base pollution with all manners of error or `bug` which causes of problems. Please check other files, and remove any linter disable.